build.gradle

```diff
@@ -231,6 +232,11 @@ spotless {
 	}
 }
 
+quality {
```

Comment:
See the pattern I put forth for quality-related overrides here: https://github.com/DataBiosphere/terra-common-lib/blob/b03c0c1f26607a4a5a9d290dc6b6af021115185f/gradle/quality.gradle

A few key points:
 - All suppressions or exclusions should have a comment with "Terra" in the line, and describing why we're making the override.
 - Omit any .xml files that aren't changed; it's extra noise and the quality plugin will by default use their managed version.